pkmodel/solution.py

```diff
@@ -14,8 +14,9 @@ class Solution:
     INPUTS
     ------------
     model: object
-
-    protocol object
+        see Model class
+    protocol: object
+        see Protocol class
 
     This class creates solution objects that contain the time interval,
```

Comment:
Perhaps say "one or more solution objects" so as to not imply that you will always get more than one?